pkg/controller/controller.go

```diff
@@ -132,15 +132,15 @@ func (c *Controller) run(ctx context.Context) error {
 
 // Called from leader elector -if configured- to start running as the leader.
 func (c *Controller) onStartedLeading(ctx context.Context) {
-	ctx, cancelNow := context.WithCancel(context.Background())
+	backgroundCtx, cancelNow := context.WithCancel(context.Background())
```

Comment:
lint complained about overriding the context

Reply:
I wish it complains about shadowing any variable...